docs/sources/source/training.rst

```diff
@@ -42,10 +42,11 @@ Use `torch.distributed.launch` package to run your script like this (assuming 8
 Example
 ~~~~~~~
 
-Please refer to the `<nemo_git_repo_root>/examples/asr/jasper.py` for comprehensive example. 
+Please refer to the `<nemo_git_repo_root>/examples/asr/jasper.py` for a comprehensive example. 
 It builds one train DAG and up to three validation DAGs to evaluate on different datasets.
 
-Assuming, you are working with Volta-based DGX, you can run training like this:
+If you are working with a +
```

Comment:
Why do we have a '+' sign here?

Reply:
`<nemo_git_repo_root>/examples/asr/jasper.py` doesn't compile for some reason

Reply:
> The plus sign in part of the Sphinx syntax in the .rst file that helps produce an effect in the document.

Unfortunately, this is what is rendered in the final doc
![image](https://user-images.githubusercontent.com/4763352/70750679-b2bb6180-1ce3-11ea-9d1f-a60d656768ea.png)
Notice the '+' is there

Reply:
Correcting.